### PR TITLE
Styling tweaks conform the requirements

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard1**
## Summary

- Add correct escalation bubbles
- Add white background to escalation map
- Legend is now rendered above the map on mobile

## Motivation

Styling request

## Detailed design

n/a

## Drawbacks

n/a

## Alternatives

n/a

## Unresolved questions

n/a

### Governance

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License
      https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt
      and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### CODE REVIEW

File: src/scss/layout.scss [hunk 1]
```diff
@@ -60,6 +60,14 @@
   margin-bottom: $default-margin * 2;
 }
 
+.index-article {
+  background: white;
+  border-radius: 5px;
```
Comment (VWSCoronaDashboard4) [inline src/scss/layout.scss:65]: there are 2x border radius

File: src/scss/layout.scss [hunk 2]
```diff
@@ -60,6 +60,14 @@
...
+  padding: 2em 2em 2em 2em;
```
Comment (VWSCoronaDashboard4) [inline src/scss/layout.scss:66]: You can use shorthand

```suggestion
  padding: 2em;
```

File: src/scss/layout.scss (hunk 1 above)
Comment (VWSCoronaDashboard1) [reply src/scss/layout.scss:65]: Oops

File: src/scss/layout.scss [hunk 3]
```diff
@@ -60,6 +60,14 @@
...
+  border-radius: 5px;
+  margin: -1rem 0 0 0;
```
Comment (VWSCoronaDashboard4) [inline src/scss/layout.scss:68]: You can use shorthand

```suggestion
  margin: -1rem 0 0;
```

File: src/scss/layout.scss [hunk 4]
```diff
@@ -60,6 +60,14 @@
   margin-bottom: $default-margin * 2;
 }
 
+.index-article {
+  background: white;
+  border-radius: 5px;
+  padding: 2em 2em 2em 2em;
+  border-radius: 5px;
```
Comment (VWSCoronaDashboard4) [inline src/scss/layout.scss:67]:
```suggestion
```

**Review by VWSCoronaDashboard4 — APPROVED** [review]

**Pull request merged**